taskcat/s3_sync.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, s3_client, bucket, prefix, path, acl="private"):
         """Syncronizes local file system with an s3 bucket/prefix
 
         """
+        self.upload_issue = False
```

Comment:
no need for this from what I can see

Reply:
Refresh and see https://github.com/aws-quickstart/taskcat/pull/172/commits/84b21055dbc5ad2153a9521c93252488eda05e0f